### PR TITLE
Feature/23 footer

### LINKED ISSUE #23: Create footer
- [ ] Create footer component
- [ ] Responsiveness

image

### PR DESCRIPTION
**Pull request by annymal**

### CODE REVIEW

File: client/src/components/ui/Container.tsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { PropsWithChildren } from 'react'
+
+export type TContainerProps = PropsWithChildren
+
+/** Basic layout container component */
+export const Container = ({ children }: TContainerProps) => {
```
Comment (Lordpluha) [inline client/src/components/ui/Container.tsx]: You can use Container: FC<PropsWithChildren>

File: client/src/components/ui/footer/Footer.module.scss [hunk 2]
```diff
@@ -0,0 +1,158 @@
+.footer {
+	@apply bg-[#161B1D] text-white absolute bottom-0 w-full font-["Rubik"]
```
Comment (Lordpluha) [inline client/src/components/ui/footer/Footer.module.scss]: use variables

Comment (Lordpluha) [reply client/src/components/ui/footer/Footer.module.scss]: use relative position

Comment (Lordpluha) [reply client/src/components/ui/footer/Footer.module.scss]: use font from tailwind

File: client/src/components/ui/footer/Footer.module.scss [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+	font-sans z-50;
+}
+
+.footer__information {
+	@apply mt-[24px] mx-auto relative px-[32px] pb-[58px] max-w-[1280px];
```
Comment (Lordpluha) [inline client/src/components/ui/footer/Footer.module.scss]: try not to use px

File: client/src/components/ui/Container.tsx
Comment (Lordpluha) [inline client/src/components/ui/Container.tsx:1]: Delete this file

File: client/.storybook/theme/assets/icon-age.png
Comment (Lordpluha) [inline client/.storybook/theme/assets/icon-age.png:1]: move this file to needed location in module in ui/ slice

File: client/src/pages/Layout.tsx [hunk 4]
```diff
@@ -1,6 +1,7 @@
 import { Outlet } from 'react-router-dom'
 
 import { Header } from '@widgets/Header'
+import { Footer } from '@widgets/footer'
```
Comment (Lordpluha) [inline client/src/pages/Layout.tsx]: Big letter start naming

**Review by Lordpluha — CHANGES_REQUESTED** [review]

File: client/src/widgets/footer/ui/Footer.tsx
Comment (annymal) [inline client/src/widgets/footer/ui/Footer.tsx:1]: не работает картинка по такому пути.
не знаю, почему.

**Review by Lordpluha — APPROVED** [review]

**Pull request merged**